libs/schemas/src/entities/thread.schemas.ts

```diff
@@ -81,3 +81,11 @@ export const Comment = z.object({
 
   Address: Address.optional(),
 });
+
+export const ThreadVersionHistory = z.object({
+  id: PG_INT.optional(),
+  thread_id: PG_INT,
+  address: z.string(),
```

Comment:
would be nice to add a description to the schema... guessing the address is the user updating the thread (owner or super admin)